### ISSUE TITLE
Hide Modal Background

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1686]
For a lot of (but not all) modals, the background is not hidden. This means that elements in the background are still focusable. This is confusing for keyboard and screen reader users as they can select controls that are not relevant to the current context, it's harder to find the controls in the modal, and the screen reader reads out non-relevant information. We should ensure that when any modal is opened the background is hidden. Also, focus should be shifted to the modal.
